ajax/libs/opus-recorder/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "opus-recorder",
+  "version": "4.1.0",
+  "description": "A library for recording opus encoded audio",
+  "homepage": "https://github.com/chris-rudmin/opus-recorder",
+  "author": "Chris Rudmin",
+  "license": "MIT",
+  "filename": "recorder.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/chris-rudmin/opus-recorder.git"
+  },
+  "npmName": "opus-recorder",
```

Comment:
Let's use git auto-update to fetch non-minified files and more versions.
```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/chris-rudmin/opus-recorder.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/!(*.wasm)"
        ]
      },
      {
        "basePath": "dist-unminified",
        "files": [
          "**/!(*.wasm|*.wast)"
        ]
      },
      {
        "basePath": "",
        "files": [
          "<pattern_for_old_versions>"
        ]
      }
    ]
  }
```

Please fill out the pattern for the old versions v0.1 ~ v0.4.4

Reply:
Can't you just include everything in `dist` for each version? Anyway, maybe the author feels like adding the pattern for the old versions, it seems overkill to me and since I'm not the lib maintainer: cc @chris-rudmin

Reply:
@thijstriemstra Since release v0.6.0 all required files should be in the `dist` folder.

Reply:
@chris-rudmin Thanks for help!
@thijstriemstra Just like Chris says :slightly_smiling_face:
I ignored v0.5.0, so we would need a pattern for this version.
But my recommendation here is that you can just fill out the pattern for the latest version. (So just leave my last review). I can help you manually add all the old versions which have different file structure.